docs/project/dogfooding.md

```diff
@@ -140,107 +135,233 @@ make it self-contained by adding a RuntimeIdentifier (RID).
-  <packageSources>
-    <add key="local coreclr" value="D:\git\corefx\artifacts\packages\Debug" /> <!-- Change this to your own output path -->
-    <add key="dotnetcore-feed" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
-  </packageSources>
-</configuration>
-```
-Be sure to correct the path to your build output above.
-
-You also have the alternative of modifying the Nuget.Config
-at `%HOMEPATH%\AppData\Roaming\Nuget\Nuget.Config` (Windows) or `~/.nuget/NuGet/NuGet.Config` (Linux) with the new location.
-This will allow your new runtime to be used on any 'dotnet restore' run by the current user.
-Alternatively you can skip creating this file and pass the path to your package directory using
-the -s SOURCE qualifier on the dotnet restore command below. The important part is that somehow
-you have told the tools where to find your new package.
-
-Once have made these modifications you will need to rerun the restore and publish as such.
-
-```
-dotnet restore
-dotnet publish
-```
-Now your publication directory should contain your local built CoreFx binaries.
-
-#### 3 - Consuming subsequent code changes by overwriting the binary (Alternative 1)
-
-To apply changes you subsequently make in your source tree, it's usually easiest to just overwrite the binary in the publish folder. Build the assembly containing your change as normal, then overwrite the assembly in your publish folder and running the app will pick up that binary. This relies on the fact that all the other binaries still match what is in your bin folder so everything works together.
-
-#### 3 - Consuming subsequent code changes by rebuilding the package (Alternative 2)
-
-This is more cumbersome than just overwriting the binaries, but is more correct.
-
-First note that Nuget assumes that distinct builds have distinct version numbers.
-Thus if you modify the source and create a new NuGet package you must give it a new version number and use that in your
-application's project. Otherwise the dotnet.exe tool will assume that the existing version is fine and you
-won't get the updated bits. This is what the Minor Build number is all about. By default it is 0, but you can
-give it a value by setting the BuildNumberMinor environment variable.
-```bat
-    set BuildNumberMinor=3
-```
-before packaging. You should see this number show up in the version number (e.g. 4.6.0-dev.18626.1).
-
-Alternatively just delete the existing copy of the package from the Nuget cache. For example on
-windows (on Linux substitute ~/ for %HOMEPATH%) you could delete
-```bat
-     %HOMEPATH%\.nuget\packages\Microsoft.Private.CoreFx.NETCoreApp\4.6.0-dev.18626.1
-     %HOMEPATH%\.nuget\packages\runtime.win-x64.microsoft.private.corefx.netcoreapp\4.6.0-dev.18626.1
-```
-which should make `dotnet restore` now pick up the new copy.
+### Nightly builds table
+
+Columns containing CoreCLR in name of the column have links to download runtimes that are backed by the CoreCLR VM.
```

Comment:
There are no plans to created shared runtime packages based on Mono. I do not think we need to be future-proofing here - it would just create confusion.

Reply:
Ok